bricks/permission_handler/__brick__/{{project_name.snakeCase()}}/lib/utils/wrappers/permission_wrapper.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:permission_handler/permission_handler.dart'
+    as permission_handler;
+
+abstract class PermissionWrapper {
+  Future<bool> requestCameraPermission();
+
+  Future<bool> isCameraPermissionDenied();
+
+  Future<bool> isCameraPermissionPermanentlyDenied();
+}
+
+class PermissionWrapperImpl extends PermissionWrapper {
+  @override
+  Future<bool> requestCameraPermission() {
+    return permission_handler.Permission.camera
+        .request()
+        .then((value) => value == permission_handler.PermissionStatus.granted);
```

Comment:
```suggestion
        .then((status) => status == permission_handler.PermissionStatus.granted);
```

Reply:
@hoangnguyen92dn thanks, updated in [de4d201](https://github.com/nimblehq/flutter-templates/pull/203/commits/de4d201fb66a46cf82bfb847c66ddeeefc349f8b)